api.md

```diff
@@ -40,7 +43,7 @@ component will get an [ApiDataBinding](#apidatabinding) or [ApiDataBinding](#api
 
 **Returns**
 
-**Function** Function to wrap your component
+**Function** Function to wrap your component, which adds a prop for each binding and an apiDataActions prop with type [Actions](#actions).
```

Comment:
Kun je nog iets duidelijker maken dat de apiDataActions prop wordt toegevoegd? Het is nu niet heel inzichtelijk wat precies het resultaat is van de HOC. Misschien onder in het codevoorbeeld even een comment toevoegen over deze prop.